IX Bid Adapter: add gpp support flag to ix bidder doc

shahinrahbariasl opened the pull request:
## 🏷 Type of documentation
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [ ] new bid adapter
- [x] update bid adapter
- [ ] new feature
- [ ] text edit only (wording, typos)
- [ ] bugfix (code examples)
- [ ] new examples

Adds GPP support to IX bidder doc.

PBJS: https://github.com/prebid/Prebid.js/blob/master/modules/ixBidAdapter.js#L722
Prebid Server: adapter passes regs to exchange.

For more information contact shahin.rahbariasl@indexexchange.com.

bretg commented:
@shahinrahbariasl - I don't see a code PR corresponding to this doc PR. Where's the evidence that your PBJS/PBS adapters accept and forward the GPP params?

shahinrahbariasl commented:
> @shahinrahbariasl - I don't see a code PR corresponding to this doc PR. Where's the evidence that your PBJS/PBS adapters accept and forward the GPP params?

Hi @bretg , I just updated the description. Here's PBJS [code](https://github.com/prebid/Prebid.js/blob/master/modules/ixBidAdapter.js#L722) in our adapter.
And in Prebid Server, our adapter passes through the regs object to exchange so I just added a test making sure that's the behaviour: https://github.com/prebid/prebid-server/pull/2521
Actually if you can help merging that one as well it'll be awesome, thank you!

bretg commented:
Cool, thanks for the thorough response and for adding the test. Will let the engineering team handle that review.
